File: tests/fixtures/java-sinks/repo/app/Reports.java
package app;

class Reports {
    @Gov(level = "protected", reason = "export boundary", owner = "security-reviewer", sink = true)
    void download() {}

    @Gov(level = "protected", reason = "direct review only", owner = "security-reviewer")
    void directOnly() {}

    void registryTarget() {}
}

File: tests/fixtures/indirect-impact/gov-opt-in/base/app/reports.py
from app.utils import check_permission


def gov(**kwargs):
    def decorator(func):
        return func
    return decorator


@gov(level="protected", reason="PII export", owner="security-reviewer", sink=True)
def download_report(user):
    check_permission(user)
    return "report"


@gov(level="protected", reason="Direct edit only", owner="security-reviewer")
def direct_only(user):
    check_permission(user)
    return "direct"
